Source/player.cpp

```diff
@@ -2073,6 +2073,22 @@ int PlayerStruct::GetBaseAttributeValue(CharacterAttribute attribute) const
 	}
 }
 
+int PlayerStruct::GetCurrentAttributeValue(CharacterAttribute attribute) const
+{
+	switch (attribute) {
+	case CharacterAttribute::Dexterity:
+		return this->_pDexterity;
+	case CharacterAttribute::Magic:
+		return this->_pMagic;
+	case CharacterAttribute::Strength:
+		return this->_pStrength;
+	case CharacterAttribute::Vitality:
+		return this->_pVitality;
+	default:
+		app_fatal("Unsupported attribute");

```

Comment:
I don't think this is needed since you have a case for all values?

Reply:
I literally copy/pasted existing code ...

Reply:
was just thinking out loud :shrug: